test-case/multiple-pipeline-capture.sh

```diff
@@ -48,20 +48,17 @@ tplg=${OPT_VALUE_lst['t']}
 func_pipeline_export $tplg
 [[ ${OPT_VALUE_lst['s']} -eq 1 ]] && func_lib_start_log_collect
 
-max_count=0
 # get the min value of TPLG:'pipeline count' with Case:'pipeline count'
-[[ $PIPELINE_COUNT -gt ${OPT_VALUE_lst['c']} ]] && max_count=${OPT_VALUE_lst['c']} || max_count=$PIPELINE_COUNT
+max_count=${OPT_VALUE_lst['c']}
 func_lib_setup_kernel_last_line
 
 # now small function define
 declare -A APP_LST DEV_LST
-APP_LST['playback']='aplay'
-DEV_LST['playback']='/dev/zero'
+APP_LST['playback,both']='aplay'
+DEV_LST['playback,both']='/dev/zero'
 APP_LST['capture,both']='arecord'
 DEV_LST['capture,both']='/dev/null'
```

Comment:
@fredoh9 I couldn't understand this logic, it seems conflict with case comments
so you will catch the same pipeline, this behavior is right?

Reply:
@Bin-QA My idea is both pipeline is counted as 1 but actually it's two pipelines. To maximize utilization, we should use both pipeline for both playback and capture. Same answer for below comment too.